### ISSUE TITLE
Remove unused CSS

### ISSUE THREAD

**Issue by bengineer19** [nuxt-community/vuetify-module#342]
**Module version**
1.11.0

**Describe the bug**
Even with treeshaking enabled, Vuetify includes a large amount of unneeded CSS when the nuxt project is built. 

**To Reproduce**
Build any nuxt-vuetify project, open Chrome devtools, go to Sources, open Coverage tab, observe how much CSS (and JS) is unused.

By default this is included in vendors.js as main.sass, but setting `extractCSS: true` in webpack config pulls out the CSS into a separate file, where the inefficiency can be clearly viewed. I have included a pic of one of my projects in the screenshots section below. The CSS which is inlined in HTML files also has plenty of waste.

This occurs for the quickstart create-nuxt-app default page, and the default code sandbox https://codesandbox.io/s/nuxtjs-vuetify-v0k7i.

**Expected behavior**
Would like only the necessary CSS to be shipped to the browser.

**Screenshots**
![image](https://user-images.githubusercontent.com/6606828/84172364-d7667b00-aa73-11ea-8418-2d780d4602ec.png)

**Comment by mathe42** [comment]
You are in Development mode? Try it in production => optimized build.

**Comment by bengineer19** [comment]
No, this is after running `nuxt build` then `nuxt start`. NODE_ENV is set to production and `nuxt build` logs "production build".

**Comment by mathe42** [comment]
OK strange...

**Comment by bengineer19** [comment]
Here is the coverage of the default app created from the quickstart, running after `$ npm run build && npm run start`.
In `nuxt.config.js` I set `treeShake: true` and `extractCSS: true`

__To be clear, this appears to suggest that every project using this module ships 283kB of CSS to the browser, of which over 90% is unused for most applications.__

![image](https://user-images.githubusercontent.com/6606828/84376856-a1430b80-abd9-11ea-9d9e-911624e09718.png)

**Comment by yohane55** [comment]
when I set `extractCSS: true`, it usually increases the bundle size. try it without it.

**Comment by little3201** [comment]
does it occurred when use vuetify only with vue?

**Comment by Guebbit** [comment]
Interested and following. I'd love to include some of vuetify plugins and optimize everything

**Comment by Vout** [comment]
Same here, just made a fresh project, cleaned out all the default stuff inside index.vue so nothing of Vuetify should be loaded, did a build and it serves a 314KiB css file with all the Vuetify css in it.

Vuetify related stuff inside my nuxtjs.config.js,

`
// Vuetify module configuration (https://go.nuxtjs.dev/config-vuetify)
  vuetify: {
    treeShake: true,
  },

  // Build Configuration (https://go.nuxtjs.dev/config-build)
  build: {
    // analyze: true,
    extractCSS: true,
  },
`

You would expect a small css bundle size but it's huge instead. Anyone who has a solution for this?

**Comment by lthh89vt** [comment]
I ran to the same issue too. Only use one VDatePicker component however the building includes the whole `main.sass` css which is where the 300kb comes from.

**Comment by mathe42** [comment]
`main.scss` is allways included.

**Comment by mathe42** [comment]
See also #382

**Comment by lthh89vt** [comment]
Not sure if we have any potential fix for this. `main.scss` is big and including some reset css which affect the global styling
